roles/openshift_node/tasks/upgrade/rpm_upgrade.yml

```diff
@@ -14,6 +21,7 @@
     - result.stdout is search(".*No package .* available.*") or result is failed
   vars:
     openshift_node_upgrade_rpm_list:
+      - "{{ openshift_service_type }}{{ openshift_pkg_version | default('') }}"
```

Comment:
openshift_pkg_version should always be defined, let's clean up these unnecessary defaults.

Reply:
Ok